packages/x-date-pickers/src/YearPicker/PickersYear.tsx

```diff
@@ -100,64 +95,64 @@ const noop = () => {};
 /**
  * @ignore - internal component.
  */
-export const PickersYear = React.forwardRef<HTMLButtonElement, YearProps>(function PickersYear(
-  props,
-  forwardedRef,
-) {
-  const {
-    autoFocus,
-    className,
-    children,
-    disabled,
-    onClick,
-    onKeyDown,
-    value,
-    tabIndex,
-    onFocus = noop,
-    onBlur = noop,
-    ...other
-  } = props;
-  const ref = React.useRef<HTMLButtonElement>(null);
-  const refHandle = useForkRef(ref, forwardedRef as React.Ref<HTMLButtonElement>);
-  const wrapperVariant = React.useContext(WrapperVariantContext);
+export const PickersYear = React.forwardRef<HTMLButtonElement, PickersYearProps>(
+  function PickersYear(props, forwardedRef) {
+    // TODO: add 'useThemeProps' once the component class names are aligned
```

Comment:
Yup, we can't easily use `useThemeProps` as it will complain about function name being different from `component` name, hence I decided to leave adding `defaultProps` support with a breaking change adjusting the component name at the same time